pingrid.py

```diff
@@ -405,8 +405,19 @@ def parse_color_item(vs: List[BGRA], s: str) -> List[BGRA]:
         rs = [parse_color(s[1:])]
     elif s[-1] == "]":
         n = int(s[:-1])
-        assert 1 < n <= 256 and len(vs) != 0
-        rs = [vs[-1]] * (n - 1)
+        assert 1 < n <= 256 and len(vs) >= 2
+        first = vs[-2]
+        last = vs[-1]
+        vs = vs[:-2]
+        rs = [
+            BGRA(
+                first.blue + (last.blue - first.blue) * i / n,
+                first.green + (last.green - first.green) * i / n,
+                first.red + (last.red - first.red) * i / n,
+                255
+            )
+            for i in range(n)
+        ]
```

Comment:
Here's what I see:

![image](https://user-images.githubusercontent.com/766406/161109696-11e19c4b-addb-4769-9b2c-9dfb5a3851f8.png)

Are you sure you have the latest version of this branch? As of an hour ago it was df84821b5d271c3b9bf91a0cb69c3f924d99ddaa, then I pushed 0a932bf8f3ed59fbcff5c58b27889085b45e2808 but that just adds a comment, so for testing either is fine.

Apologies if I rebased/revised after you had started reviewing. I generally try to avoid that.

Reply:
So I've just pulled and indeed
Updating df84821..0a932bf
and so that doesn't change anything and I have the 'max' error and a somewhat truncated colorbar
![Screen Shot 2022-03-31 at 1 06 12 PM](https://user-images.githubusercontent.com/42619415/161111081-fc80be4c-1319-4645-b3b5-6bcc6139cad5.png)
.

Reply:
In mine, the callback is indeed returning the string `"90"` rather than the number `90`, but the front end doesn't mind. My best guess is that we have different versions of the Colorbar component, which would come from having different versions of dash-leaflet.  What does this print in your environment?
```
conda list | grep dash-leaflet
```